src/librustc_error_codes/error_codes/E0730.md

```diff
@@ -1,6 +1,6 @@
 An array without a fixed length was pattern-matched.
 
-Example of erroneous code:
+Erroneous code example:
```

Comment:
It's now plural ;)

Reply:
There is still only one erroneous code example. The other code examples are not "erroneous". :p